docker/package.json

```diff
@@ -10,7 +10,7 @@
     "homepage": "https://pulumi.io",
     "repository": "https://github.com/pulumi/pulumi-docker",
     "dependencies": {
-        "@pulumi/pulumi": "^0.15.0",
+        "@pulumi/pulumi": "^0.15.3-dev",
```

Comment:
We shouldn't be committing dev build dependencies, right?

Reply:
Yes - master branches need to track dev dependencies to carry fixes up stack.  That's been our approach for last few months.  We cannot release a new non-dev build of this repo until we release a non-dev build of `@pulumi/pulumi`.  Said another way - the `@release/*` branches cannot ever have `dev` build dependencies.